src/v2/guide/components-registration.md

```diff
@@ -172,7 +172,7 @@ export default {
 }
 ```
 
-Just to support relatively little markup in a template:
+Sólo para soportar relativamente poco markup en una plantilla:
```

Comment:
@sxcamacho @FLarra Are we translating `markup`?

Reply:
Hi guys, we are using `marcado` for `markup '. If you think it is better to keep the 'markup' in English, we can change the rest of the occurrences (there are only some cases).

Reply:
@Tintef @sxcamacho is ok for me to use 'marcado' if you used it before.

Reply:
I think `marcado` is not a really good translation for `markup`. Let's keep markup and change the rest